great_expectations/rule_based_profiler/helpers/runtime_environment.py

```diff
@@ -106,9 +142,8 @@ def _build_specific_domain_type_directives(
                 domain_type_directives_keys(key)
             )
             domain_type_directives.directives[domain_type_directives_key] = value
-        except ValueError as e:
-            raise ge_exceptions.ProfilerExecutionError(
-                message=f'Unknown property "{key}" in "{domain_type_directives_keys}"; {e} was raised.'
-            )
+        except ValueError:
+            # Skip every directive that is not defined key in some "RuntimeEnvironmentDomainTypeDirectivesKeys" member.
+            pass
```

Comment:
Could we please add a `logger.debug()` statement here?

Reply:
@cdkini I believe that this message will not be valuable.  This is one of those very rare cases when the exception handling has to help implement the logic (i.e., this is a normal case, for which we do not want warnings in notebooks).